src/Native/Windows/clrcompression/zlib-intel/README

```diff
@@ -0,0 +1,115 @@
+ZLIB DATA COMPRESSION LIBRARY
```

Comment:
Should we add something in the README, FAQ, ChangeLog, Index, etc. about zlib-intel and the changes that it brings?